app/templates/components/ui-table/cell/admin/tickets/cell-amount.hbs

```diff
@@ -0,0 +1 @@
+{{record.event.paymentCurrency}} {{record.amount}}
```

Comment:
No record is available. pass in the required properties as valuePath and extraValuePaths

Reply:
@niranjan94 This is the sample screenshot of the progress for now:

![image](https://user-images.githubusercontent.com/21087061/60771830-f5f91f00-a10a-11e9-9da6-8126e7be0463.png)
The `record` seems to be working here :thinking: , am I doing something wrong?

Reply:
https://github.com/fossasia/open-event-frontend/blob/development/app/templates/components/tables/default.hbs#L36-L37

Record does not refer to the row. it refers to the specific value for a cell. Never attempt to use the whole row from within a cell. 

Also @CosmicCoder96 , please rename `record` to value or something. Causes a confusion. Record sounds more like it refers to the whole model record.